app/mailers/asynchronous_notify.rb

```diff
@@ -0,0 +1,36 @@
+require 'celluloid'
```

Comment:
I think the name of this will would be better as: `BroadcastMailer`. What do you think?